misaligned.c

```diff
@@ -16,6 +18,16 @@ int printColorMap() {
 int main() {
     int result = printColorMap();
     assert(result == 25);
+    assert(result == 0);
+    assert(result == 1);
+    assert(result == 2);

```

Comment:
Do you expect result to be 25, 0,1,2,3,4,6,8 etc at the same time ?
Isnt it wrong expectation from production code?

Reply:
Yes, I just tried will change that in the coming assignment